cloverage/test/cloverage/instrument_test.clj

```diff
@@ -75,7 +75,7 @@
     (t/is (= 1 (both-defn 1)))))
 
 (t/deftest test-exclude-calls
-  (let [form    '(inc 100) 
+  (let [form   '(inc 100)
```

Comment:
FWIW, I'm definitely with you here; this was totally correct the first time round. Sorry (I inherited the config and this project lacks someone with the resources to maintain it properly :/)

Reply:
D'oh, hold on.

Reply:
Ok, it was just the trailing whitespace for that line. Amended the commit to just address that.